docs/proposals/job_controller.md

```diff
@@ -0,0 +1,226 @@
+**Abstract**
+
+First basic implementation proposal for a Job controller. Several exiting issues were already created regarding that particular subject:
+
+    Distributed CRON jobs in k8s #2156
+    Job Controller #1624
+
+Several features also already exist that could be used by external tools to trigger batch execution on one pod within k8 cluster.
+
+    Create a standalone pod (not linked to any existing replication controllers)
+    Execute a new process within a given container from a running pod.
+
+**Motivation**
+
+The main goal is to provide a new pod controller type, called Job, with the following characteristics:
```

Comment:
To be consistent with k8s, we'll probably name this a JobController, and single running instance of a piece of work will be a Job.